src/js/background/identityState.js

```diff
@@ -36,6 +37,19 @@ const identityState = {
     return Object.assign({}, tab);
   },
 
+  async getCookieStoreIDuuidMap() {
+    const containers = {};
+    const containerInfo = await identityState.storageArea.area.get();
+    for(const key of Object.keys(containerInfo)) {
```

Comment:
what's an example `key` value here? Could we make the variable name more specific like `urlKey`?

Reply:
It just all the keys for the configuration in storage. I just called it configKey.  Too vague?